src/tools/map/making/PointFileReaderWriter.java

```diff
@@ -1,4 +1,4 @@
-package games.strategy.util;
+package tools.map.making;
```

Comment:
Maybe better to leave this one in games/strategy for now since its used by GameData.